harvard-lts/CURIOSity
# Docker ignore

**Update docker ignore to exclude docker files and docker composes**
* * *


# What does this Pull Request do?
Adds Dockerfile* and docker-compose* to docker-ignore

# How should this be tested?

We're going to check when we deploy as locally these would be copied even with the ignore

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties
